src/components/views/messages/RedactedBody.tsx

```diff
@@ -14,21 +14,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import React, {useContext} from "react";
-import {MatrixClient} from "matrix-js-sdk/src/client";
-import {MatrixEvent} from "matrix-js-sdk/src/models/event";
+import React, { useContext } from "react";
+import { MatrixClient } from "matrix-js-sdk/src/client";
+import { MatrixEvent } from "matrix-js-sdk/src/models/event";
 import { _t } from "../../../languageHandler";
 import MatrixClientContext from "../../../contexts/MatrixClientContext";
-import {formatFullDate} from "../../../DateUtils";
+import { formatFullDate } from "../../../DateUtils";
 import SettingsStore from "../../../settings/SettingsStore";
+import { MatrixClientPeg } from "../../../MatrixClientPeg";
 
 interface IProps {
     mxEvent: MatrixEvent;
+    isExporting: boolean;
 }
 
-const RedactedBody = React.forwardRef<any, IProps>(({mxEvent}, ref) => {
-    const cli: MatrixClient = useContext(MatrixClientContext);
-
+const RedactedBody = React.forwardRef<any, IProps>(({mxEvent, isExporting}, ref) => {
+    let cli: MatrixClient = useContext(MatrixClientContext);
+    // As context doesn't propagate during export, we'll have to explicitly declare
```

Comment:
Use a `<MatrixClientContext.Provider` around your mount for exporting and then it would.